server/src/main/scala/org/scalaexercises/evaluator/services.scala

```diff
@@ -1,30 +1,34 @@
 /*
- * scala-exercises - evaluator-server
- * Copyright (C) 2015-2016 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ *  scala-exercises - evaluator-server
+ *  Copyright (C) 2015-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
  */
 
 package org.scalaexercises.evaluator
 
-import monix.execution.Scheduler
+import cats.effect.{ConcurrentEffect, ContextShift, ExitCode, IO, IOApp, Timer}
+import cats.implicits._
+import codecs._
+import coursier.util.Sync
+import coursier.interop.cats._
+import io.circe.generic.auto._
+import io.circe.syntax._
 import org.http4s._
 import org.http4s.dsl._
+import org.http4s.headers.Allow
 import org.http4s.server.blaze._
 import org.log4s.getLogger
+import org.http4s.syntax.kleisli.http4sKleisliResponseSyntax
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
 object services {
 
   import EvalResponse.messages._
-  import codecs._
-  import io.circe.generic.auto._
 
-  private val logger = getLogger
-
-  implicit val scheduler: Scheduler = Scheduler.io("scala-evaluator")
-
-  val evaluator = new Evaluator(20 seconds)
+  def evaluator[F[_]: ConcurrentEffect: ContextShift: Timer: Sync] = new Evaluator[F](20 seconds)
```

Comment:
My concern here is that we would be creating a new evaluator instance for every request